src/Solver.cpp

```diff
@@ -624,11 +625,14 @@ void Solver::countFissionableFSRs() {
   log_printf(INFO, "Counting fissionable FSRs...");
 
   /* Count the number of fissionable FSRs */
-  _num_fissionable_FSRs = 0;
+  int num_fissionable_FSRs = 0;
+#pragma omp parallel for reduction(+:num_fissionable_FSRs)
   for (int r=0; r < _num_FSRs; r++) {
     if (_FSR_materials[r]->isFissionable())
-      _num_fissionable_FSRs++;
+      num_fissionable_FSRs++;
   }
+
+  _num_fissionable_FSRs = num_fissionable_FSRs;
```

Comment:
Nope, I'll add one since. I did mess this up in one of my commits and was not computing the number of fissionable FSRs correctly so this will be a good test to have.